plugins/persistence.lua

```diff
@@ -0,0 +1,170 @@
+local PLUGIN = PLUGIN
+
+PLUGIN.name = "Persistence"
+PLUGIN.desc = "Saves persisted entities through restarts."
+PLUGIN.author = "Zoephix"
+PLUGIN.entities = PLUGIN.entities or {}
+PLUGIN.blacklist = PLUGIN.blacklist or {}
+
+-- Copy blacklist from the restrictions plugin
+local restrictions = nut.plugin.list["restrictions"]
```

Comment:
restrictions plugin?
Either integrate that as well or remove 'restrictions' reliances